test/integration/quic_http_integration_test.cc

```diff
@@ -1900,9 +1965,6 @@ TEST_P(QuicHttpIntegrationTest, UsesPreferredAddressDualStack) {
   });
 
   initialize();
-  quic::QuicTagVector connection_options{quic::kSPAD};
```

Comment:
@RyanTheOptimist these tests are currently failing. I thought I'd be able to remove this bit with the new flag. Does it only apply to the serverside, and not the clientside, in QUICHE?

Reply:
refreshReloadableFlags() might not be called on test client. You can force the flag to be true here
https://github.com/envoyproxy/envoy/blob/77f9577c26d675d921759cc51ae8bc3db92dd2e5/source/common/quic/platform/quiche_flags_impl.cc#L28-L38 and remove the RUNTIME_GUARD as QUICHE flags can also flipped via Envoy runtime.

Please add a test to verify if the flag is overridden, Envoy will require SPAD.

Reply:
Got it; fixed the default by adding it here. However, because this is a QUICHE_PROTOCOL_FLAG the auto mechanism for updating it without additional envoy work does not apply, so I still have the envoy runtime guard.

Reply:
Ah, RTDS doesn't work for QUICHE_PROTOCOL_FLAG.